doc/userguide/install.rst

```diff
@@ -75,40 +75,102 @@ For full features, also add::
 
 Rust support::
 
-  rustc, cargo
+  rustc, cargo, cbindgen
 
-  Not every distro provides Rust packages yet. Rust can also be installed
-  directly from the Rust project itself::
+  Rust packages can be found in package managers but some distros
+  don't provide or provide outdated Rust packages.
+  In case of insufficient version you can install Rust directly
+  from the Rust project itself::
```

Comment:
The paragraph here provides additional info in case of troubles but suggests installing it primarily from package managers. Copypaste examples are below